lua/telescope/mappings.lua

```diff
@@ -75,9 +77,14 @@ local get_next_id = function()
   return _mapping_key_id
 end
 
-local assign_function = function(prompt_bufnr, func)
+local assign_function = function(prompt_bufnr, func, mode, keybind)
   local func_id = get_next_id()
 
+  -- if type(func) == "table" then
```

Comment:
No longer needed?

Reply:
Thanks for catching that :sweat_smile: apologies :)